pkg/service/node.go

```diff
@@ -160,8 +162,37 @@ func (n *NodeService) NodeGetVolumeStats(context.Context, *csi.NodeGetVolumeStat
 	panic("implement me")
 }
 
-func (n *NodeService) NodeExpandVolume(context.Context, *csi.NodeExpandVolumeRequest) (*csi.NodeExpandVolumeResponse, error) {
-	panic("implement me")
+func (n *NodeService) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandVolumeRequest) (*csi.NodeExpandVolumeResponse, error) {
+	klog.Infof("Resizing filesystem mounted on %s", req.VolumePath)
```

Comment:
Same as above,
I think that we should add a check here for the disk size and return codes.OK if the size is already larger than or equal to req.CapacityRange.RequiredBytes, according to the CSI docs[1]

```
This operation MUST be idempotent. If a volume corresponding to the specified volume ID is already larger than or equal to the target capacity of the expansion request, the plugin SHOULD reply 0 OK.
```

[1] https://github.com/container-storage-interface/spec/blob/master/spec.md#nodeexpandvolume